gemrb/plugins/GUIScript/GUIScript.h

```diff
@@ -23,13 +23,7 @@
 
 // NOTE: Python.h has to be included first.
 
-#if defined(WIN32) && defined(_DEBUG)
-#undef _DEBUG
 #include <Python.h>
-#define _DEBUG
-#else
-#include <Python.h>
-#endif
```

Comment:
Probably needed for older python, but it doesn't matter any more. 👍🏾

Reply:
Sadly—as many headers do when they see MSVC—Python.h links to the export lib with a pragma, and it causes a mismatch with Python's debug DLL when fooled this way.

Reply:
So does this still work with both release and debug builds?

Reply:
Yes, but now the debug build uses the debug Python library.

Reply:
Ok, but this is code for everywhere, so I wonder if it still builds with MSYS2. Looking at their packages, it seems they build only the release version. @MarcelHB, can you check if this PR compiles?

Reply:
I've tried to check it myself, but MSYS2 and MinGW toolchains seem to always build the release version, which works.

Reply:
Can confirm, builds and starts.

Reply:
Great, thanks!